src/components/profile-background.tsx

```
@@ -0,0 +1,167 @@
+                    className="absolute -bottom-1 -right-1 z-30 flex size-5 items-center justify-center rounded-full bg-background text-xs"
+                    title={`Frame: ${presetFrame?.name || "Custom"}`}
+                >
+                    {frameEmoji}
+                </div>
+            )}
+        </div>
+    );
+}
+
+function getFrameEmoji(framePreset?: string): string | null {
+    if (!framePreset) return null;
```

ESLint

[error] Expected { after 'if' condition.

(curly)